chpl-src/announcing-chapel-2.6.chpl

```diff
@@ -0,0 +1,463 @@
+/*
+
+{{<todo>}} Consider removing following sidebar depending on status at
+time of publication {{</todo>}}
+
+  The Chapel community is pleased to announce the release of Chapel
+  2.6!  As usual, you can [download and
+  install](https://chapel-lang.org/download/) this new version in a
+  {{<sidenote "right" "variety of formats">}}Note that some formats
+  may not be immediately available on the day of the
+  release...{{</sidenote>}}, including Spack, Docker, Homebrew,
+  various Linux package managers, and good-old source tarballs.
+
+  In this article, we'll introduce some of the highlights of Chapel
+  2.6, including:
+
+  * Chapel's new module for [dynamically loading
+    libraries](#dynamic-loading-support) and calling into them
+
+  * Improvements when [debugging Chapel programs](#debugging-enhancements)
+
+  * Improved [unit
+    testing](#unit-testing-improvements-for-mason-and-vscode) with
+    Mason and VSCode
+
+  * Numerous [documentation and
+    `chpldoc`](#documentation-improvements) improvements
+
+  * Improvements to the capabilities of the [Dyno compiler
+    front-end](#improvements-to-the-dyno-compiler-front-end)
+
+
+  In addition to the above features, each of which is covered in more
+  detail below, other highlights of Chapel 2.6 include:
+
+  * Improved support for using [address
+    sanitizers](https://chapel-lang.org/docs/2.6/usingchapel/debugging/sanitizers.html)
+    with Chapel programs
+
+  * Ongoing capability improvements to Chapel's tools, such as
+    [VSCode](https://chapel-lang.org/docs/2.6/usingchapel/editor-support.html#vscode),
+    the [`chplcheck`](https://chapel-lang.org/docs/2.6/tools/chplcheck/chplcheck.html) linter,
+    [`chpl-language-server`](https://chapel-lang.org/docs/2.6/tools/chpl-language-server/chpl-language-server.html), and [`chapel-py`](https://chapel-lang.org/docs/2.6/tools/chapel-py/chapel-py.html)
+
+  * Flexibility improvements to the
+    [Homebrew](https://chapel-lang.org/download/#homebrew) and [Linux
+    package](https://chapel-lang.org/download/#linux) releases of
+    Chapel, including support for:
+    - Chapel's cpu-as-device GPU emulation mode
+    - support for multi-locale executions with Homebrew installs
+    - support for both the LLVM or C-based back-ends from Linux
+      packages
+
+    In addition, where past Linux packages supported a binary download
+    per Chapel configuration, the 2.6 release bundles all
+    configurations for a given OS/platform into a single binary.
+
+  For a much more complete list of changes in Chapel 2.6, see the
+  [CHANGES.md](https://github.com/chapel-lang/chapel/blob/release/2.6/CHANGES.md)
+  file.  And as always, thanks to [everyone who
+  contributed](https://github.com/chapel-lang/chapel/blob/release/2.6/CONTRIBUTORS.md)
+  to version 2.6!
+
+
+  ### Dynamic Loading Support
+
+  This release of Chapel offers improved support for calling into
+  dynamically loaded libraries.  This feature was introduced in a
```

Comment:
I would just say "calling dynamically loaded" and drop "into"

Reply:
I wasn't convinced (I don't think of libraries as being something you call), but David proposed "calling procedures from dynamically loaded libraries” which I think is even better if that's OK with you.